code/modules/clothing/suits/cloaks.dm

```diff
@@ -89,3 +89,41 @@
 	heat_protection = HEAD
 	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
 	resistance_flags = FIRE_PROOF | ACID_PROOF
+
+/obj/item/clothing/neck/cloak/skill_reward
+	var/associated_skill_path
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE
+
+/obj/item/clothing/neck/cloak/skill_reward/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>You notice a powerful aura about this cloak, suggesting that only the truly experienced may wield it.</span>"
+
+/obj/item/clothing/neck/cloak/skill_reward/equipped(mob/user, slot)
+	..()
+	if (user.mind?.get_skill_level(associated_skill_path) < SKILL_LEVEL_LEGENDARY)
```

Comment:
what happens if someone else (who is or isn't a legendary [skill]er) tries to put the cloak on your shoulders?

Reply:
they shouldn't be able to hold it
i think equipped calls when someone puts something on you

Reply:
yes it does. and they cant even pick it up